reccmp/isledecomp/compare/analyze.py

```diff
@@ -68,6 +69,21 @@ def create_analysis_floats(db: EntityDb, img_id: ImageId, binfile: PEImage):
                 )
 
 
+def create_seh_entities(db: EntityDb, img_id: ImageId, binfile: PEImage):
+    """Create entities for the SEH (structured exception handling)
+    handler and funcinfo struct. For images without a relocation table,
+    this will allow us to replace the addresses for both items."""
+    with db.batch() as batch:
+        for handler_addr, funcinfo in find_eh_handlers(binfile):
+            # Using names derived from symbols in .cpp.s generated asm.
+            batch.set(
```

Comment:
Just curious: Do these entities have no type? Is that something we support, or that has any use to us?

If there is some default type they are set to, I'd prefer to have that explicit.